Sources/TuistLoader/Loaders/SwiftPackageManagerGraphLoader.swift

```diff
@@ -165,6 +167,22 @@ public final class SwiftPackageManagerGraphLoader: SwiftPackageManagerGraphLoadi
             externalProjects: externalProjects
         )
     }
+
+    private func validatePackageResolved(at path: AbsolutePath) {
+        let savedPackageResolvedPath = path.appending(components: [
+            Constants.SwiftPackageManager.packageBuildDirectoryName,
+            Constants.DerivedDirectory.name,
+            Constants.SwiftPackageManager.packageResolvedName,
+        ])
+        let savedData = try? fileHandler.readFile(savedPackageResolvedPath)
+
+        let currentPackageResolvedPath = path.appending(component: Constants.SwiftPackageManager.packageResolvedName)
+        let currentData = try? fileHandler.readFile(currentPackageResolvedPath)
```

Comment:
Let's throw an 
```suggestion
        if !fileHandler.exists(currentPackageResolvedPath) {
          SwiftPackageManagerGraphGeneratorError.installRequired
        }
        let currentData = try fileHandler.readFile(currentPackageResolvedPath)
```

let's get rid of the `try?` calls that could swallow actually unexpected errors

Reply:
If the `Package.swift` doesn't list any remote dependencies, SPM doesn't create a `Package.resolved`. So there are cases where we wouldn't want to throw, like `fixtures/app_with_previews`.

I did change it to check if the file exists and don't swallow the error if it does.